src/recordlinker/schemas/pii.py

```diff
@@ -45,6 +52,42 @@ def __str__(self):
         """
         return self.value
 
+class Race(enum.Enum):
+    """
+    Enum for the Race field.
+    """
+
+    AMERICAN_INDIAN = "american indian or alaska native"
+    ASIAN = "asian"
+    BLACK = "black or african american"
+    HAWAIIAN = "native hawaiian or other pacific islander"
+    WHITE = "white"
+    OTHER = "other"
+    ASKED_UNKNOWN = "asked but unknown"
+    UNKNOWN = "unknown"
+
+    def __str__(self):
+        """
+        Return the value of the enum as a string.
+        """
+        return self.value
```

Comment:
I'm thinking maybe the keys will be a better representation of the data in the database than the values.  Its not as descriptive, but it better aligns to current values like those in the spreadsheet that Krishna sent over.  Thoughts?

Reply:
Yeah that makes sense to me. I can do the same thing for `Gender` as well